src/views/Peers.tsx

```diff
@@ -216,6 +233,16 @@ export const Peers = () => {
         )
     }
 
+
+    function handleSwitchSSH(record: PeerDataTable, checked: boolean) {
+        const peer = {
+            id: record.id,
+            ssh_enabled: checked,
+            name: record.name
+        } as Peer
```

Comment:
Notice this one now. To prevent issues in the future with API validating payloads, we should use a proper Type for requests as this resource's request is one of the few that doesn't contain all fields on PUT requests.